Fabric.Identity.API/Models/Domain/ClientDomainModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Fabric.Identity.API.Models.Domain
+{
+    public class ClientDomainModel : IdentityServer4.EntityFramework.Entities.Client, ISoftDelete, ITrackable
```

Comment:
These models should be in the SqlServer namespace because they are specific to that implementation. the IS4.Models are the domain models.

Reply:
good call. i will move them